alf/bin/train.py

```diff
@@ -121,6 +121,13 @@ def _setup_device(rank: int = 0):
 
 def _setup_remote_configs_if_needed():
     """Preconfig some configurations for remote training and unrolling.
+
+    There will also be a global config flag named '_USER.remote_training' set in
```

Comment:
_USER should be used for user defined configs. Perhaps we can put this into TrainerConfig.